tests/splines/matrix.cpp

```diff
@@ -52,14 +55,16 @@ void check_inverse(ddc::DSpan2D matrix, ddc::DSpan2D inv)
         for (std::size_t j(0); j < N; ++j) {
             double id_val = 0.0;
             for (std::size_t k(0); k < N; ++k) {
-                id_val += matrix(i, k) * inv(j, k);
+                id_val += matrix(i, k) * inv(k, j);
```

Comment:
Was it working because the matrix is symmetric ?

Reply:
The matrix being symmetric makes it work regardless of the choice of convention indeed. However the change here is aligned to the change of convention brought with the current MR: the elements of a matrix `A_ij` are identified with indices `(i, j)` (it was previously `(j,i)`)

Reply:
Do you mean `A_ij` was identified by `get/set_element(j, i, a_ij)` ? I am not sure about that

Reply:
No sorry, not the matrix A, the multi-rhs b.

ie. here https://github.com/CExA-project/ddc/blob/main/include/ddc/kernels/splines/matrix.hpp#L122

Reply:
Call here: https://github.com/CExA-project/ddc/pull/444/files#diff-deb6768d88f9dcadb445a3de0cfff4717aef343c81c34c4f3bff6ae4e3653117L115

Reply:
I agree